### PR TITLE
`setupSkyLabels` accepts a node

### PR DESCRIPTION
**Pull request by seanpdoyle**
Allows callers to scope the setup function to a DOM node more specific
than `document`

### CODE REVIEW

File: src/sky-labels.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
-var setupSkyLabels = function () {
-  $(document).on("focus blur", ".sky-label", addOrRemoveHasTextClass);
-  $(document).on("focus", ".sky-label", addFocusedClass);
-  $(document).on("blur", ".sky-label", removeFocusedClass);
-  $(document).ready(hideLabelsIfInputHasText);
```
Comment (seanpdoyle) [inline src/sky-labels.js:5]: Should this stay `document`?

**Comment by paulcsmith** [comment]
@seanpdoyle I don't quite understand the use case for this. It seems like if you want to only apply it to a certain node, then you could only use the sky-labels classes in that node?

**Comment by seanpdoyle** [comment]
**TL;DR** setting up `document` wide event handlers doesn't work in Ember.

@paulcsmith I recently ported Sky Labels to an [Ember addon](https://github.com/thoughtbot/ember-sky-labels/blob/9061abc190950cedb5ffd3a3b61a9ae47ba5d81d/addon/components/sky-label.js).

Unfortunately, due to the way these event handlers are declared, none of the JS logic was re-usable.

We could probably close this, as I don't think is way to write this library that would make this code more portable and reusable across frameworks.

**Comment by paulcsmith** [comment]
@seanpdoyle Ok sounds good. Sorry it's been so tricky to port over. I'll have to look up why Ember doesn't work with document wide event handlers. It seems like that shouldn't be an issue unless it's stopping propagation of events or something :(
